input/STARSS/method_feedback_starss.in

```diff
@@ -0,0 +1,266 @@
+    HeI_density   = 0.3   * 4.0*1.2E-24;
+    HeII_density  = 1e-10 * 4.0*1.2E-24;
+    HeIII_density = 1e-10 * 4.0*1.2E-24;
+    e_density     = 1e-10 * 4.0*1.2E-24;           # uniform mass density for ICs
+    metal_fraction = 1e-2*0.012;
+
+    position = [0.5,0.5,0.5]; # unigrid corner (or one block)
+
+    temperature = 100.0;              # in K
+                                      #   particles are still local to each block and are kicked away from
+                                      #   boundaries if NxN feedback zone crosses a block boundary
+    star_mass  = 1000.0;                # particle mass in Msun
+  }
+
+}
+
+Output {
+    list = ["de","te","vx"];
+     de {
+         dir = [ "FEEDBACK_TEST_%04d", "cycle" ];
+         field_list = [ "density" ];
+         image_ghost = false;
+         image_size = [ 512, 512 ];
+         image_type = "data";
+         name = [ "de-%04d.png", "cycle" ];
+         schedule {
+             start = 0; #1;
+             step = 1;
+             var = "cycle";
+         };
+         type = "image";
+     };
+     te {
+         dir = [ "FEEDBACK_TEST_%04d", "cycle" ];
+         field_list = [ "total_energy" ];
+         image_ghost = false;
+         image_size = [ 512, 512 ];
+         image_type = "data";
+         name = [ "te-%04d.png", "cycle" ];
+         schedule {
+             start = 0; #1;
+             step = 1;
+             var = "cycle";
+         };
+         type = "image";
+     };
+     vx {
+         dir = [ "FEEDBACK_TEST_%04d", "cycle" ];
+         field_list = [ "velocity_x" ];
+         image_ghost = false;
+         image_size = [ 512, 512 ];
+         image_type = "data";
+         name = [ "vx-%04d.png", "cycle" ];
+         schedule {
+             start = 0; #1;
+             step = 1;
+             var = "cycle";
+         };
+         type = "image";
+     };
+     vy {
+         dir = [ "FEEDBACK_TEST_%04d", "cycle" ];
+         field_list = [ "velocity_y" ];
+         image_ghost = false;
+         image_size = [ 512, 512 ];
+         image_type = "data";
+         name = [ "vy-%04d.png", "cycle" ];
+         schedule {
+             start = 0; #1;
+             step = 1;
+             var = "cycle";
+         };
+         type = "image";
+     };
+     mesh {
+         colormap = [ 0.000000000000000, 0.000000000000000, 0.000000000000000, 0.000000000000000, 0.000000000000000, 1.000000000000000, 0.000000000000000, 1.000000000000000, 1.000000000000000, 0.000000000000000, 1.000000000000000, 0.000000000000000, 1.000000000000000, 1.000000000000000, 0.000000000000000, 1.000000000000000, 0.000000000000000, 0.000000000000000 ];
+         dir = [ "FEEDBACK_TEST_%04d", "cycle" ];
+         image_reduce_type = "max";
+         image_size = [ 1025, 1025 ];
+         image_type = "mesh";
+         name = [ "mesh-%02d.png", "count" ];
+         schedule {
+	     start = 0; #1;
+             step = 1;
+             var = "cycle";
+         };
+         type = "image";
+     };
```

Comment:
It might be helpful to delete these or add comments to explain that these aren't actually being used (to avoid confusing new users). But, it's totally up to you

Reply:
deleted